src/components/MobileTable/MobileTable.tsx

```diff
@@ -29,8 +29,8 @@ const MobileTable: FC<MobileTableProps> = ({
       <MobileTableRow
         key={index}
       >
-        {columns?.map((column) => (
-          <div key={`column-${column.field || ''}`}>
+        {columns?.map((column, index) => (
+          <div key={`column-${column.field || ''}-${index}`}>
```

Comment:
the main idea behind "key" in react is to make it unique. By adding index you just silence linter and ignore keys feature. In case of reorder - you will rerender this.

Not a big deal for this part, but let's keep it clean.

Reply:
WIthout index there were tons of warnings about non-unique keys in the console, when you resize screen with a table. And these are keys for cells within 1 row.